scripts/docbuild.rb

```diff
@@ -30,5 +30,6 @@ def execute(cmd)
 
 $log.info("Copying docs to fileshare...")
 FileUtils.rm_rf DOCS_FILESHARE
+FileUtils.mkdir_p DOCS_FILESHARE
```

Comment:
@Fryguy any update on this ? Can we merge this for now at least until there's a fix in asciibinder land ? (maybe nhr can help there). Thanks.